app/controllers/admin/merchants_controller.rb

```diff
@@ -7,9 +7,11 @@ def update
     merchant = Merchant.find(params[:merchant_id])
     if merchant.active?
       merchant.disable
+      merchant.items.each { |item| item.deactivate }
```

Comment:
Should this live inside the merchant model?  It seems to be a part of the merchant.disable method